ome/ome-release
# Add logic to symlink promoted virtualbox artifacts (develop)

To test this PR:

promote OMERO-trunk, OMERO-trunk-ice34 and OMERO-trunk-virtualbox jobs
run OME-merge-release-develop and check the virtualbox artifacts have been properly symlinked
